pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1440,6 +1440,18 @@ type GatewaySpec struct {
 	HostNetwork *bool `json:"hostNetwork,omitempty"`
 }
 
+// EndpointAddress is a tuple that describes a single IP address or host name. This is a subset of
+// Kubernetes's v1.EndpointAddress.
+// +structType=atomic
+type EndpointAddress struct {
+	// The IP of this endpoint.
+	// +optional
+	IP string `json:"ip" protobuf:"bytes,1,opt,name=ip"`
```

Comment:
Are the protobuf annotations really necessary here?

Reply:
I'm not sure, but it seemed reasonable to keep them to make sure the CRD that gets generated is compatible with the old type.

Reply:
either way should be fine, they should be ignored since we're not using protobuf internally with rook